dev-util/shellcheck/shellcheck-0.9.0.ebuild

```
@@ -28,3 +28,8 @@ RDEPEND=">=dev-haskell/aeson-1.4.0:=[profile?] <dev-haskell/aeson-2.2:=[profile?
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-3.0.0.0
 "
+
+src_install() {
+	haskell-cabal_src_install
+	doman "${FILESDIR}/${PN}.1"
+}
```

Would this be better?

```suggestion
DEPEND="${RDEPEND}
	>=dev-haskell/cabal-3.0.0.0
"
BDEPEND="man? ( || (
	app-text/pandoc
	app-text/pandoc-bin
) )"

src_compile() {
	haskell-cabal_src_compile
	if use man; then
		pandoc -s -f markdown-smart -t man "${PN}.1.md" -o "${PN}.1" || exit
	fi
}

src_install() {
	haskell-cabal_src_install
	doman "${S}/${PN}.1"
}
```